### ISSUE TITLE
Batch run mode

### ISSUE THREAD

**Issue by mwang87** [Arseha/peakonly#9]
Is it possible to classify features just on the commandline in linux rather than the GUI?

**Comment by macraild** [comment]
Not officially.

But this script  [https://github.com/macraild/peakonly/blob/master/peakonly_batch.py](https://github.com/macraild/peakonly/blob/master/peakonly_batch.py) reimplements a simplified version of processing_utils.runner.FilesRunner._batch_run, to give access to the basic functionality from the command line. Its an ugly hack, but in my hands gives identical results to the GUI.

Minimally, 
`python peakonly_batch.py --mzML *.mzML`
should work as long as the peakonly install dir is on your PYTHON_PATH, but check
`python peakonly_batch.py -h`
for other options.

**Comment by soerendip** [comment]
I get: 

    python peakonly_batch.py --mzML /data/Metabolomics/MS-formats/MTBLS1569/mzML/QC_*

> peakonly_batch.py [-h] [--mzML mzMLfiles] [--mz-deviation MZ_DEVIATION] [--min-ROI-length MIN_ROI_LENGTH] [--max-zeros MAX_ZEROS] [--min-peak-length MIN_PEAK_LENGTH]
                         [--model-dir directory] [--csv csvFiles] [--png directory]
peakonly_batch.py: error: unrecognized arguments: /data/Metabolomics/MS-formats/MTBLS1569/mzML/QC_1.mzML /data/Metabolomics/MS-formats/MTBLS1569/mzML/QC_2.mzML /data/Metabolomics/MS-formats/MTBLS1569/mzML/QC_3.mzML /data/Metabolomics/MS-formats/MTBLS1569/mzML/QC_4.mzML

**Comment by macraild** [comment]
Yes, I forgot that *nix and Windows handle command-line wildcards differently.
I've updated the script to fix.
